modules/islandora_collection_permissions/includes/handler.class.inc

```diff
@@ -191,20 +191,24 @@ class IslandoraCollectionPermissionsHandler {
     }
 
     $object = islandora_object_load($pid);
-    $related = array();
-
-    foreach (array('isMemberOfCollection', 'isMemberOf') as $pred) {
-      foreach ($object->relationships->get(FEDORA_RELS_EXT_URI, $pred) as $relation) {
-        $related[] = $relation['object']['value'];
+    if (is_object($object)) {
+      $related = array();
+
+      foreach (array('isMemberOfCollection', 'isMemberOf') as $pred) {
+        foreach ($object->relationships->get(FEDORA_RELS_EXT_URI, $pred) as $relation) {
+          $test = islandora_object_load($relation['object']['value']);
+          if (is_object($test)) {
+            $related[] = $relation['object']['value'];
+          }
+        }
       }
-    }
 
-    $related = array_unique($related);
-    sort($related);
+      $related = array_unique($related);
+      sort($related);
 
-    return $related;
+      return $related;
+    }
   }
-
```

Comment:
This blank line should be added back in to adhere to Drupal coding standards.

Reply:
👍